src/Common/isLocalAddress.cpp

```diff
@@ -111,6 +102,23 @@ bool isLocalAddress(const Poco::Net::IPAddress & address)
         }
     }
 
+    return false;
+}
+
+bool isLocalAddress(const Poco::Net::IPAddress & address)
+{
+    /** 127.0.0.1 is treat as local address unconditionally.
+      * ::1 is also treat as local address unconditionally.
+      *
+      * 127.0.0.{2..255} are not treat as local addresses, because they are used in tests
+      *  to emulate distributed queries across localhost.
```

Comment:
So it will not protect from specifying `127.0.0.2` in Keeper configuration?

Reply:
Really... I'll fix it.